src/XMLexport.cpp

```diff
@@ -203,7 +203,7 @@ void XMLexport::writeModuleXML(const QString& moduleName, const QString& fileNam
 
     auto future = QtConcurrent::run(this, &XMLexport::saveXml, fileName);
     auto watcher = new QFutureWatcher<bool>;
-    QObject::connect(watcher, &QFutureWatcher<bool>::finished, [=]() { mpHost->xmlSaved(fileName); });
+    QObject::connect(watcher, &QFutureWatcher<bool>::finished, mpHost, [=]() { mpHost->xmlSaved(fileName); });
```

Comment:
Is this sufficient, `mpHost` is a `QPointer<Host>` so will get reset to a `nulllptr` if the Host instance is destroyed, but does that behaviour (the value *stored* in the `QPointer` carry across into the lamda?) - would it be any better/safer/more robust if this (and the same sort of thing in the other two) was revised to:

    QObject::connect(watcher, &QFutureWatcher<bool>::finished, mpHost, [=]() {if (mpHost) {mpHost->xmlSaved(fileName);}});